### ISSUE TITLE
log clustering and deep log

### ISSUE THREAD

**Issue by Rufaida94** [logpai/loglizer#73]
Is it possible to write a version for these two demos that can work without having a label file?

from my understanding of the log clustering method, that might not be possible because it needs the label file for the knowledge base, correct?

**Comment by tanay1997** [comment]
Yeah, I am facing the same issue? I want to run deep log without labels, is it possible? @Rufaida94 did you find a solution? Thanks

**Comment by Rufaida94** [comment]
@tanya199

No, I produced my own labels but even after that I faced other issues with DeepLog and decide to move on to a different thing since it seems like the authors are not responding.

**Comment by Rufaida94** [comment]
@tanya1997 
No, I produced my own labels but even after that I faced other issues with DeepLog and decide to move on to a different thing since it seems like the authors are not responding.

**Comment by Rufaida94** [comment]
@tanay1997

No, I produced my own labels but even after that I faced other issues with DeepLog and decide to move on to a different thing since it seems like the authors are not responding.
